payments/invoice/redirect.md

```diff
@@ -244,6 +245,7 @@ next_href="seamless-view" next_title="Next: Seamless View" %}
 [abort]: /payments/invoice/after-payment#abort
 [after-payment]: /payments/invoice/after-payment
 [callback]: /payments/invoice/other-features#callback
+[paymentUrl]: /checkout/other-features#payment-url-1
```

Comment:
Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6181597)